en/chromebook_setup/instructions.md

```diff
@@ -61,6 +61,77 @@ This bottom area is your terminal. You can use the terminal to send instructions
 to the remote Cloud 9 computer. You can resize that window to make it a bit
 bigger.
 
+#### Glitch.com Cloud IDE
+
+1. Go to [Glitch.com](https://glitch.com/)
+2. Sign up for an account (https://glitch.com/signin) or use your Github account if you have one (see Github instructions below).
+3. Click _New Project_ and choose _Hello Webpage_
+4. Click Terminal button (on the left side of the window)
+
+When using Glitch.com as your Cloud IDE, we do not create a virtual environment but we create the following files manually :
+
+{% filename %}glitch.json{% endfilename %}
+```
+{
+  "install": "pip3 install -r requirements.txt --user",
+  "start": "bash start.sh",
+  "watch": {
+    "throttle": 1000
+  }
+}
+```
+
+{% filename %}requirements.txt{% endfilename %}
+```
+Django>=2.2.4
+```
+
+{% filename %}.bash_profile{% endfilename %}
+```
+alias python=python3
+alias pip=pip3
+```
+
+
+{% filename %}start.sh{% endfilename %}
+```
+chmod 600 .bash_profile
+pip3 install -r requirements.txt --user
+python3 manage.py makemigrations
+python3 manage.py migrate
+python3 manage.py runserver $PORT
+```
+
+Once these files are created, open a terminal (in the tools menu at the bottom) and execute the command ```refresh```.
+
+The initial restarting of the Glitch project should fail (complaining that the ```manage.py``` file is missing), but do not worry about it at this stage, the tutorial will fix this as soon as you create your first Django project.
+
+In order to see detailed error messages, you can activate Django debug logs for your Glitch application.
+Simply add the following at the end of the ```mysite/settings.py``` file.
+
+{% filename %}mysite/settings.py{% endfilename %}
+```python
+LOGGING = {
```

Comment:
`mysite/settings.py` hasn't yet been created at this point in the tutorial.

Reply:
Indeed, I updated the instructions to run the "first Django project" command first, followed by a glitch `refresh` command, so that the file exists.